custom-service-adapter/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>knotx.io.tutorial</groupId>
+  <artifactId>custom-service-adapter</artifactId>
+  <version>1.0.0</version>
+  <name>Custom service adapter</name>
+
+  <properties>
+    <!-- the main verticle class name -->
+    <main.verticle>com.cognifide.knotx.launcher.KnotxStarterVerticle</main.verticle>
+    <file.encoding>UTF-8</file.encoding>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>1.8</java.version>
+    <handlebars.version>4.0.5</handlebars.version>
+    <vertx.version>3.3.3</vertx.version>
+    <rxjava.version>1.1.9</rxjava.version>
+    <knotx.version>1.0.0</knotx.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.knotx</groupId>
+      <artifactId>knotx-core</artifactId>
+      <version>${knotx.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.knotx</groupId>
+      <artifactId>knotx-adapter-common</artifactId>
+      <version>${knotx.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.1.3</version>
+    </dependency>
+
+    <!-- CORE -->
```

Comment:
You don't need to add vertx and rxjava to the dependencies (except jdbc) as vert.x provides it through transitive depndencies